### PR TITLE
verify and document minimum node/yarn version

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:
- tests that module can be installed with minimum node version
- documents minimum node version in package.json
- fixes some issue with verify package on TSC/CJS (adds webpack devDependencies)

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [x] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [x] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-507707](https://oktainc.atlassian.net/browse/OKTA-507707)

### Downstream Monolith Build:

https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&branch=d16t-okta-signin-widget-0c66e40-62fa6dcb

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
     "not ie < 11",
     "not IE_Mob 11"
   ],
+  "engines": {
```
Comment (shuowu-okta) [inline package.json:23]: do we need this to overcome some CI issues? otherwise, maybe minor/major version is a better to to add this kind of package field

**Review by shuowu-okta — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (aarongranick-okta) [reply package.json:23]: This is formallly documenting/enforcing our CURRENT minimum node version. We plan to up the minimum version in the next major release

**Comment by lesterchoi-okta** [comment]
Changes seem ok, but unable to verify locally. Hitting a grunt issue.

**Comment by aarongranick-okta** [comment]
> Changes seem ok, but unable to verify locally. Hitting a grunt issue.

you will need to do a release build first: `yarn build:release`. Then `mv node_modules node_modules2`. Then within any of the 3 package test folders run `yarn` then `yarn test`

**Pull request merged**